.travis.yml

```diff
@@ -4,28 +4,25 @@ language: haskell
 git:
   depth: 5
 
-cabal: "2.4"
+cabal: "3.0"
 
 cache:
   directories:
   - "$HOME/.cabal/store"
-  
 
 matrix:
   include:
-  - ghc: 8.4.4
   - ghc: 8.6.5
-  
+  - ghc: 8.8.1
```

Comment:
is it enough?

Reply:
Do you mean, is it enough to test only the latest 2 GHC versions? I guess for now it's okay. Otherwise, it could be a maintenance nightmare, since the libraries are too old. And we might even stop adding new GHC versions if it becomes impossible to build other libraries with them.